tests/test_events_api.py

```diff
@@ -0,0 +1,137 @@
+from os import environ
+from typing import Optional
+from unittest.mock import MagicMock, patch
+
+from lbz.aws_boto3 import Boto3Client
+from lbz.events.api import BaseEvent, EventAPI
+
+
+class MyTestEvent(BaseEvent):
+    type = "MY_TEST_EVENT"
+
+
+class TestBaseNewEvent:
+    def test_base_new_event(self) -> None:
```

Comment:
The initialization is tested multiple times below - I don't see value in testing it separately.

Much better would be to focus on tests that bring real value - I mentioned examples in the above comments 😉